vm/ubpf_jit_x86_64.c

```diff
@@ -314,6 +313,8 @@ translate(struct ubpf_vm* vm, struct jit_state* state, char** errmsg)
             emit_alu64_imm32(state, 0x81, 5, RSP, 8);
         }
 
+        state->pc_locs[i] = state->offset;
```

Comment:
Just confirming that I am reading this correctly: In the case that the instruction we are about to JIT is the target of a local function call, we are setting the offset of the code associated with eBPF instruction `i` to be *after* the stack adjustment code? Then (see below), we adjust to target the stack adjustment code when we actually `call `that local eBPF function (as opposed to `jump `to it). 

If that is correct, I like!!

Reply:
Correct. We also now compute the size of the stack adjustment code.